src/schemas/21P-530/schema.js

```diff
@@ -40,17 +44,25 @@ let schema = {
           type: 'string'
         }
       }
-    }
+    },
+    claimantAddress: schemaHelpers.getDefinition('address'),
+    claimantEmail: {
+      type: 'string',
+      format: 'email'
+    },
+    claimantPhone: schemaHelpers.getDefinition('usaPhone'),
   }
 };
 
 [
+  ['address'],
```

Comment:
can you change this to `['address', 'claimantAddress']` and remove claimantAddress above?